### PR TITLE
Dehardcode status change skill assosiation

### PR DESCRIPTION
**Pull request by Asheraf**
### Pull Request Prelude

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR may be closed if the above-mentioned criteria are not fulfilled.

**Issues addressed:** 

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

File: src/map/status.c [hunk 1]
```diff
@@ -176,630 +165,465 @@ static void initChangeTables(void)
...
-	status->set_sc( NPC_STUNATTACK    , SC_STUN      , SCB_NONE );
-	status->set_sc( NPC_SLEEPATTACK   , SC_SLEEP     , SCB_NONE );
-	status->set_sc( NPC_POISON        , SC_POISON    , SCB_DEF2|SCB_REGEN );
-	status->set_sc( NPC_CURSEATTACK   , SC_CURSE     , SCB_LUK|SCB_BATK|SCB_WATK|SCB_SPEED );
-	status->set_sc( NPC_SILENCEATTACK , SC_SILENCE   , SCB_NONE );
-	status->set_sc( NPC_WIDECONFUSE   , SC_CONFUSION , SCB_NONE );
-	status->set_sc( NPC_BLINDATTACK   , SC_BLIND     , SCB_HIT|SCB_FLEE );
-	status->set_sc( NPC_BLEEDING      , SC_BLOODING  , SCB_REGEN );
-	status->set_sc( NPC_POISON        , SC_DPOISON   , SCB_DEF2|SCB_REGEN );
+	status->set_sc(SC_STONE     , SCB_DEF_ELE|SCB_DEF|SCB_MDEF );
```
Comment (4144) [inline src/map/status.c:168]: probably change style on changed lines?

Comment (Asheraf) [reply src/map/status.c]: updated

File: src/map/status.c [hunk 2]
```diff
@@ -176,630 +165,465 @@ static void initChangeTables(void)
...
+	status->set_sc(SC_BLESSING, SCB_STR | SCB_INT | SCB_DEX);
+	status->set_sc(SC_CONCENTRATION, SCB_AGI | SCB_DEX);
+	status->set_sc(SC_HIDING, SCB_SPEED);
+	status->set_sc(SC_TWOHANDQUICKEN, SCB_ASPD);
+	status->set_sc(SC_IMPOSITIO,
 #ifdef RENEWAL
-		SCB_NONE );
+		SCB_NONE);
 #else
 		SCB_WATK );
```
Comment (EPuncker) [inline src/map/status.c:195]:
```suggestion
		SCB_WATK);
```

Comment (Asheraf) [reply src/map/status.c:195]: this line was not touched by me

**Review by 4144 — APPROVED** [review]

**Review by MishimaHaruna — APPROVED** [review]

**Pull request merged**